Documentation/architecture/net-platform-standard.md

```diff
@@ -134,13 +134,21 @@ Existing PCL projects in VS2013 and VS2015 (excluding UWP targets), can only tar
 
 ### Deprecated monikers
 
-| Platform | NuGet identifier |
-| ---------| --------------- |
-| ASP.NET 5.0 on .NET Framework | aspnet50 |
-| ASP.NET 5.0 on .NET Core | aspnetcore50 |
-| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 |
-| DNX on .NET Core 5.0 | dnxcore50  |
-| Windows 8 | winrt |
+| Platform | Deprecated NuGet identifier | Current NuGet identifier |
+| ---------| --------------------------- | ------------------------ |
+| ASP.NET 5.0 on .NET Framework | aspnet50 | net46 |
+| ASP.NET 5.0 on .NET Core | aspnetcore50 | netstandard1.3 |
+| DNX on .NET Framework 4.5.1 - 4.6 | dnx451 - dnx46 | net451 - net46 |
+| DNX on .NET Core 5.0 | dnxcore50  | dotnet &rarr; dotnet5.4 &rarr; netstandard1.3 |
+| .NET Core 5.1 - 5.6 | dotnet5.1 - dotnet5.6  | netstandard1.0 - netstandard 1.5 |
+| .NET Core 5.0 | dotnet | dotnet5.4 &rarr; netstandard1.3 |
```

Comment:
Name here is wrong.  NuGet called this '.NET Platform'  Please don't use .NET Core.